st2common/bin/migrations/v3.8/st2-drop-st2exporter-marker-collections

```diff
@@ -56,7 +54,7 @@ def main():
     db_setup()
 
     try:
-        delete_marker_collections(display_prompt=not cfg.CONF.yes)
```

Comment:
pylint complained about this as the `display_prompt` kwarg doesn't exist. Oops.

I believe I copy/pasted this from the v3.5 migration script.